### PR TITLE
ARS 645: Agents for organisations, contact details

### PR DESCRIPTION
**Pull request by ymabdulahi**

### CODE REVIEW

File: test/pages/BusinessContactDetailsPageSpec.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class BusinessContactDetailsPageSpec extends PageBehaviours {
+
+  "BusinessContactDetailsPage" - {
+    def arbitraryString(gen: Gen[Char], maxSize: Int = 100): Gen[String] =
+      Gen.listOfN(maxSize, gen).map(_.mkString)
+
+    implicit val businessContactDetailsGen: Arbitrary[BusinessContactDetails] = Arbitrary(for {
+
+      name    <- arbitraryString(Gen.alphaChar)
+      phone   <- arbitraryString(Gen.numChar, maxSize = 15)
```
Comment (C-Thomp) [inline test/pages/BusinessContactDetailsPageSpec.scala:32]: Not a biggie but same here

Comment (C-Thomp) [inline test/pages/BusinessContactDetailsPageSpec.scala:32]:
```suggestion
      phone   <- arbitraryString(Gen.numChar, maxSize = 24)
```

File: test/pages/ApplicationContactDetailsPageSpec.scala [hunk 2]
```diff
@@ -26,12 +26,13 @@ class ApplicationContactDetailsPageSpec extends PageBehaviours {
...
-    implicit val applicationContactDetailsGen = Arbitrary(for {
+    implicit val applicationContactDetailsGen: Arbitrary[ApplicationContactDetails] =
+      Arbitrary(for {
 
-      name  <- arbitraryString(Gen.alphaChar)
-      phone <- arbitraryString(Gen.numChar, maxSize = 15)
-      email <- arbitraryString(Gen.asciiChar)
-    } yield ApplicationContactDetails(name, phone, email))
+        name  <- arbitraryString(Gen.alphaChar)
+        phone <- arbitraryString(Gen.numChar, maxSize = 15)
```
Comment (C-Thomp) [inline test/pages/ApplicationContactDetailsPageSpec.scala:33]:
```suggestion
        phone <- arbitraryString(Gen.numChar, maxSize = 24)
```

**Review by C-Thomp — DISMISSED** [review]
extremely minor changes but we have couple failing tests too

**Review by RawToast — APPROVED** [review]

**Review by koladeadewuyi — APPROVED** [review]
Looks good 👍 .

**Pull request merged**